interfaces/builtin/network_setup_control.go

```diff
@@ -45,6 +45,18 @@ const networkSetupControlConnectedPlugAppArmor = `
 
 /run/udev/rules.d/ rw,                 # needed for cloud-init
 /run/udev/rules.d/[0-9]*-netplan-* rw,
+
+capability net_admin,
+
+#include <abstractions/dbus-session-strict>
```

Comment:
This should be `#include <abstractions/dbus-strict>` since below you have specified `bus=system`.

Reply:
Okay, changed.